PowerPointLabs/PowerPointLabs/RemoveHighlighting.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Office.Interop.PowerPoint;
+using PowerPointLabs.Models;
+
+namespace PowerPointLabs
```

Comment:
There isn't one currently, but sure I can make one.
I'll put the other 3 classes (HighlightBulletsBackground, HighlightBulletsText, HighlightTextFragments) in as well